GoogleForCreators/web-stories-wp
# Dashboard: Telemetry Settings Opt-In Checkbox

## Summary

Add a checkbox for telemetry settings for opt-in for tracking

![image](https://user-images.githubusercontent.com/1738349/92128468-0b79df80-edc8-11ea-9d8a-c2193d8fc23e.png)


- [x] Focus support plus screen reader support
- [x] Connected to API and saves the state on check or uncheck


## Relevant Technical Choices

Follows existing pattern of adding a setting to the settings page of a separate component.

## To-do

None

## User-facing changes



## Testing Instructions

1. Enable the Settings Page for Dashboard
2. Visit Dashboard Settings
3. Scroll to Data Sharing Opt-in
4. Check it or un check it. See that your state is maintained on refresh.

The entire label should be clickable.

The Google Privacy link should open a new page _but not_ change the state.

---



#3165
